src/diffusers/models/resnet.py

```diff
@@ -40,13 +40,22 @@ def forward(self, hidden_states, output_size=None):
         if self.use_conv_transpose:
             return self.conv(hidden_states)
 
+        # Cast to float32 to as 'upsample_nearest2d_out_frame' op does not support bfloat16
+        dtype = hidden_states.dtype
```

Comment:
this works for me! Let's please add a comment to the official PyTorch issue here though